src/DonorDashboards/Tabs/EditProfileTab/AvatarRoute.php

```diff
@@ -106,4 +120,15 @@ public function handleRequest(WP_REST_Request $request)
         );
     }
 
+    /**
+     * @unreleased
+     */
+    private function avatarBelongsToCurrentUser(): bool
```

Comment:
This would be more testable if the avatar ID and the user ID were passed as parameters.

Reply:
Challenge accepted 🦾

Reply:
Updated to new method and matching test: 3019ff47